### DISCUSSION TITLE
How to ensure non-code asset files from a module are included in the bundle ?

### DISCUSSION THREAD

**Discussion by tigerinus** [pyinstaller/pyinstaller#8197] [Help] [answered]
I've encountered following issue which I filed to LangChain's repo:

- https://github.com/langchain-ai/langchain/issues/15386

Basically the `langchain` module comes with few `.txt` files, and they are not included in the bundle after running `pyinstaller`

While the issue is being triaged, any workaround to unblock?

Thanks.

**Comment by tigerinus** [comment, marked as answer]
nevermind - solution is `--collect-all`

**Reply by rokm** [reply]
If you need to collect the data files, the solution is `--collect-data`, which is more fine-grained than `--collect-all`.
